server.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Laravel - A PHP Framework For Web Artisans
+ *
+ * @package  Laravel
+ * @author   Taylor Otwell <taylor@laravel.com>
+ */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing @link tag in file comment](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)